src/imu/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.5)
+project(imu)
+
+add_definitions(-D_USE_MATH_DEFINES)
+
+include_directories(${CMAKE_SOURCE_DIR}
```

Comment:
```suggestion
```

Are you sure you need this? In this context `CMAKE_SOURCE_DIR` is the root folder of this repo, I do not think there are any include there.